resources/dicts/events/war.json

```diff
@@ -27,19 +38,46 @@
+      "o_c recently lead out a surprise attack to claim c_n territory, but to no avail.",
+      "lead_name isn't sure what to think of the whole war situation with o_c anymore, as it develops even further.",
+      "The war shows no signs of stopping, the hostility between c_n and o_c still going strong."
     ],
     "rel_down": [
       "o_c has taken some of c_n's territory.",
       "c_n has taken some of o_c's territory.",
       "War negotiations with o_c went badly at the last Gathering.",
-      "The tension between c_n and o_c is palpable, with even the smallest actions potentially leading to violence." 
+      "The tension between c_n and o_c is palpable, with even the smallest actions potentially leading to violence.",
+      "The hostile environment between c_n and o_c has been continuing to grow.",
+      "Any attempts at peace only seemed to make the whole war situation between c_n and o_c worse.",
+      "The cats of o_c and c_n are still keen on battle, with no yearn for peace shown between them.",
+      "lead_name is starting to notice how battles between o_c and c_n have been extremely aggressive recently.",
+      "Glares of unfriendliness and aggression are passed at most patrol interactions between c_n and o_c.",
+      "o_c's surprise attacks on c_n have been increasing significantly lately.",
+      "c_n's surprise attacks on o_c have been increasing significantly lately.",
+      "Even small sparks of disagreement between o_c and c_n can set off a huge battle.",
+      "c_n's cats can't stop themselves from instinctively unsheathing their claws upon encountering an o_c cat."
     ]
   },
   "conclusion_events": [
     "The war with o_c has finally ended.",
     "The cats of o_c and c_n finally agree to peace terms.",
     "A truce is called and the war between o_c and c_n is ended.",
-    "With the many injuries both sides have suffered, o_c and c_n found common ground and ended the war."
+    "With the many injuries both sides have suffered, o_c and c_n found common ground and ended the war.",
+    "The dust has finally settled of the war between o_c and c_n.",
+    "The leaders of both o_c and c_n decide that for the wellbeing of both Clans, the war should come to an end now.",
+    "After lots of bloodshed of both o_c and c_n cats, the two Clans have come to terms of peace.",
+    "The battles between o_c and c_n have ceased, the war now finally coming to an end.",
+    "o_c and c_n have had a long and complex past, but it's time for their battles to come to an end.",
+    "Many battles have been fought between o_c and c_n, but it's time for those to stop now.",
+    "The cats of both o_c and c_n have come to a realization that war isn't the best solution, deciding to end it now.",
+    "The defiantly puffed out chests of o_c and c_n have turned into tired, sulking bodies, and the cats are glad as the war finally comes to a close.",
```

Comment:
Although I love those conclusion text, I find "defiantly puffed out chests" to be slightly awkward. I suggest: 

"flashing fangs"
"defiant war cries" 
"puffed out chests" (remove "defiantly") 

I also making "...the cats are glad as the war finally comes to a close" into it's own sentence for clarity. At the moment, it reads (at first blush) as being part of a list with "tired, sulking bodies".

Reply:
i think piercing war cries would be good and yeah, wasnt sure with the whole sulking bodies thing to begin with so will do